@xen-orchestra/lite/src/components/host/network/HostPifSidePanel.vue

```diff
@@ -0,0 +1,478 @@
+import VtsCardRowKeyValue from '@core/components/card/VtsCardRowKeyValue.vue'
+import VtsConnectionStatus from '@core/components/connection-status/VtsConnectionStatus.vue'
+import VtsIcon from '@core/components/icon/VtsIcon.vue'
+import UiButton from '@core/components/ui/button/UiButton.vue'
+import UiButtonIcon from '@core/components/ui/button-icon/UiButtonIcon.vue'
+import UiCard from '@core/components/ui/card/UiCard.vue'
+import UiCardTitle from '@core/components/ui/card-title/UiCardTitle.vue'
+import UiPanel from '@core/components/ui/panel/UiPanel.vue'
+import UiTag from '@core/components/ui/tag/UiTag.vue'
+import { vTooltip } from '@core/directives/tooltip.directive'
+import { faCircle, faCopy, faEdit, faEllipsis, faStar, faTrash } from '@fortawesome/free-solid-svg-icons'
+import { useClipboard } from '@vueuse/core'
+import humanFormat from 'human-format'
+import { computed } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+const { pif } = defineProps<{
+  pif: XenApiPif
+}>()
+
+const { getByOpaqueRef: getPifMetricsByOpaqueRef, getPifCarrier } = usePifMetricsStore().subscribe()
+const { getBondsDevices, isBondMaster } = usePifStore().subscribe()
+const { getByOpaqueRef: getOpaqueRefNetwork } = useNetworkStore().subscribe()
+
+const { t } = useI18n()
+
+const addressesIp = computed(() => {
+  const ips = [pif.IP, ...pif.IPv6].filter(ip => ip)
+  return ips.length > 0 ? ips : ['-']
+})
+
+const network = computed(() => getOpaqueRefNetwork(pif.network))
+
+const networkNameLabel = computed(() => network.value?.name_label || '-')
+
+const networkPurpose = computed(() => (network.value?.purpose?.[0] ? t('on') : t('off')))
+
+const networkTags = computed(() => (network.value?.tags.length ? network.value.tags : '-'))
+
+const pifStatus = computed(() => (pif.currently_attached ? 'connected' : 'disconnected'))
+
+const physicalInterfaceStatus = computed(() => (getPifCarrier(pif) ? 'connected' : 'physically-disconnected'))
+
+const getSpeed = computed(() => getPifMetricsByOpaqueRef(pif.metrics)?.speed || 0)
+
+const getVlan = computed(() => (pif.VLAN === -1 ? t('none') : pif?.VLAN))
+
+const getNetmask = computed(() => (pif.netmask === '' ? '-' : pif.netmask))
+
+const getDNS = computed(() => (pif.DNS === '' ? '-' : pif.DNS))
+
+const getGateway = computed(() => (pif.gateway === '' ? '-' : pif.gateway))
+
+const getIpConfigurationMode = computed(() => {
+  const ipMode = pif.ip_configuration_mode
+
+  if (ipMode === 'Static') return t('static')
+  if (ipMode === 'DHCP') return t('dhcp')
+  return t('none')
+})
+
+const getMtu = computed(() => (pif.MTU === -1 ? t('none') : pif.MTU))
+
+const bondDevices = computed(() => getBondsDevices(pif))
+
+const isBond = computed(() => isBondMaster(pif))
+
+const byteFormatter = computed(() => (value: number) => {
+  const speedInBytes = value * 1000000
+
+  return humanFormat(speedInBytes, {
+    scale: 'SI',
+    unit: 'b/s',
+    maxDecimals: 2,
+  })
+})
```

Comment:
Since it is only used once in the template, maybe we can avoid calling it from the template and instead compute the result directly:
```suggestion
const byteFormatter = computed(() => {
  const speedInBytes = pifSpeed.value * 1000000

  return humanFormat(speedInBytes, {
    scale: 'SI',
    unit: 'b/s',
    maxDecimals: 2,
  })
})
```